tests/release_tests/test_launch/test_kubernetes.py

```diff
@@ -0,0 +1,166 @@
+import os
+import signal
+import subprocess
+import sys
+
+from kubernetes import client, config, watch
+from utils import get_wandb_api_key
+from wandb.sdk.launch.launch_add import launch_add
+
+
+def _create_wandb_and_aws_secrets(aws_id, aws_secret, aws_token, namespace):
+    wandb_api_key = get_wandb_api_key()
```

Comment:
In order to make testing with a service account easy, we could make the API key a CLI when calling this. Or an input when running the command